src/adapters/supabase/helpers/client.ts

```diff
@@ -64,7 +64,7 @@ export type IssueAdditions = {
   completed_at?: number;
 };
 
-const getDbDataFromIssue = (issue: Issue, additions: IssueAdditions): any => {
+const getDbDataFromIssue = (issue: Issue, additions: IssueAdditions) => {
   return {
     issue_number: issue.number,
     issue_url: issue.html_url,
```

Comment:
What is the purpose of getting the issue and url from the database when we're working directly on GitHub on issues.